server/cli/prepare.py

```diff
@@ -116,6 +120,9 @@ def load_data(data):
         if not adata._var.index.is_unique:
             click.echo("Warning: var index is not unique")
 
+        if calculate_qc_metrics:
```

Comment:
If this takes a non-trivial amount of time (test on the 1 mil dataset) move this to a function and add to the show_step dict. Otherwise, it's fine as is.